external-crates/move/crates/move-package-alt/src/package/lockfile.rs

```diff
@@ -98,7 +98,9 @@ impl<F: MoveFlavor + fmt::Debug> Lockfile<F> {
                 return Err(metadata.unwrap_err().into());
             };
 
-            let old_entry = lockfiles.published.insert(env_name.clone(), metadata);
+            let old_entry = lockfiles
+                .published
+                .insert(Located::new(env_name, file_id, Range::default()), metadata);
```

Comment:
I'm a little surprised that `Range` has a default. I assume it's 0..0? At any rate, is that the right thing to use here?